src/olympia/addons/models.py

```diff
@@ -1404,77 +1432,6 @@ def get_latest_file(self):
     def in_escalation_queue(self):
         return self.escalationqueue_set.exists()
 
-    def update_names(self, new_names):
```

Comment:
Interestingly this it the method Marketplace used to update the name on new uploads of an existing app.